src/cpp/core/include/core/r_util/RActiveSessions.hpp

```diff
@@ -490,7 +466,8 @@ class GlobalActiveSessions : boost::noncopyable
    core::FilePath rootPath_;
 };
 
-void trackActiveSessionCount(const FilePath& rootStoragePath,
+void trackActiveSessionCount(std::shared_ptr<IActiveSessionStorage> storage,
```

Comment:
```suggestion
void trackActiveSessionCount(const std::shared_ptr<IActiveSessionStorage>& storage,
```

Reply:
Actually, this function isn't actually implemented with this change, because the definition now has different parameters. Because your code still compiles & links, I think we must not be using this. Can you try deleting this function (here and in the cpp) and see if everything still compiles?